server/routers/album/album_routes.py

```
@@ -225,122 +218,116 @@ async def update_album(
 
-        return all_albums
+    return all_albums
 
 
 @router.delete("/api/photo/delete/")
-async def delete_photo(slug: str, photo_name: str):
-    with get_db() as (db, cursor):
-        cursor.execute("SELECT * FROM album WHERE slug=%s", (slug,))
-        album = cursor.fetchone()
+async def delete_photo(
+    slug: str, photo_name: str, session: Session = Depends(get_session)
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)